tests/test_jsonitem.py

```diff
@@ -75,6 +85,7 @@ def jsonitem_tuple(request):
         ('boolean', 'null', None),
         ('array', '[1,2,3]', [1, 2, 3]),
         ('array', 'null', None),
+        ('url', 'http://test.com', 'http://test.com'),
```

Comment:
This is not getting tested. The `params` above is set `range(12)`. I like this pattern: https://github.com/mesosphere/dcos-cli/blob/master/cli/tests/unit/test_marathon_validation.py#L22-L31